File: Java/FactorialNumbers.java
class FactorialNumbers{
    public static void main(String[] args){
        int n=10;
        long fact=1;
        if (n>=1){
            int i=1;
            while(i<=n){
                fact=fact*i;
                i++;
            }System.out.println("Factorial of "+n +" is : "+fact);
        }else if(n==0){
            System.out.println("Factorial of 0 is : 1");
        }else{
            System.out.println("Sorry, factorial does not exist for negative numbers");
        }
    }
}

File: Python/Factorial_Numbers.py
n = int(input("Enter a number: "))
factorial = 1
if n >= 1:
    for i in range (1,(n+1)):
       factorial = factorial * i
    print("Factorial of ",n , " is : ",factorial)
elif n==0:
    print("Factorial of 0 is : 1")
else:
    print("Sorry, factorial does not exist for negative numbers")
